node/tests/main.spec.js

```diff
@@ -64,13 +64,13 @@ describe('main', () => {
   describe('when command exists', () => {
     describe('help', () => {
       describe.each`
-        command
-        ${'-h'}
-        ${'--help'}
-        ${'help'}
-      `('when user asks for help with $command', ({ command }) => {
+        command   | rawArgs
+        ${''}     | ${['-h']}
+        ${''}     | ${['--help']}
+        ${'help'} | ${[]}
+      `('when user asks for help with command=$command, rawArgs=$rawArgs', ({ command, rawArgs }) => {
         beforeEach(async () => {
-          await mockOptionsAndRun({ command, rawArgs: [command] });
```

Comment:
Fixed the test. It was running weird commands like `env0 help help` and `env0 -h -h`.